### PR TITLE
Update to docs, examples, and tutorial links

### PR DESCRIPTION
**Pull request by michael-w-williams**
Shaun asked me to make the Tutorials links point to the tutorials repo. So in doing that, I updated the main docs and example landing pages at the same time.
- README.md: Added a link to the tutorials repo. Renamed the old tutorials series to "Getting Started Series". Not attached to that name if someone has something better.
- docs/README.md: Sorted topics alphabetically. Removed redundant links. Used an emoji to pin key topics to top of section. Used a star, could use a pushpin instead or something else.
- examples/README.md: Added links to other major doc/learning categories.
- examples/tutorial/README.md: Renamed this to "Getting Started Series".  Only linked to initial supported languages (Dropped Ruby, Rust and PHP).
- All: Added a "Learn More" section to all but the quickstart to link to doc related topics.

Let me know what you think.

Mike W.

### CODE REVIEW

**Comment by shaunsmith** [comment]
LGTM, only one comment that maybe we should replace the heading/link "client libraries" with "function developer kits" in docs/README.md?

File: docs/README.md [hunk 1]
```diff
@@ -4,38 +4,44 @@
 
 If you are a developer using Fn through the API, this section is for you.
 
-* [Quickstart](https://github.com/fnproject/fn)
-* [Usage](usage.md)
-* [Writing functions](writing.md)
-* [Testing functions](testing.md)
-* [Hot functions](hot-functions.md)
+* :star: [Quickstart](https://github.com/fnproject/fn)
+* :star: [Hot functions](hot-functions.md)
```
Comment (treeder) [inline docs/README.md]: Why are these starred?

Comment (michael-w-williams) [reply docs/README.md]: Trying to pin important topics to the top of the list. So I thought an icon might help identify them. I can use another icon or remove the emoji altogether. Let me know what you think.

**Comment by michael-w-williams** [comment]
Added a change to docs/readme.md. No icons and categories added. Feel free to update as needed.

Mike W.

**Comment by michael-w-williams** [comment]
I did the rebase. But there is still a conflict.  Need to keep Open, Packaging, Setting up. Delete API, Object, FAQ

**Comment by carimura** [comment]
github makes it easy. you gonna resolve the conflict? I can if you want

**Comment by michael-w-williams** [comment]
I will give it a shot.

**Comment by michael-w-williams** [comment]
ok. Conflict resolved.

**Comment by carimura** [comment]
will review today thx

**Pull request merged**